packages/elements/src/list/helpers/list-renderer.ts

```diff
@@ -32,6 +39,7 @@ export class ListRenderer extends Renderer {
       el.label = composer.getItemPropertyValue(item, 'label') as string;
       el.subLabel = composer.getItemPropertyValue(item, 'subLabel') as string;
       el.value = composer.getItemPropertyValue(item, 'value') as string;
+      el.id = `${this.key}-${el.value}`;
```

Comment:
Do we need `key` here? Render is always rendered in shadowDom, is not it?

Reply:
No, the items are in light DOM